solidity/random-beacon/test/utils/snapshot.ts

```diff
@@ -0,0 +1,15 @@
+import { ethers } from "hardhat"
```

Comment:
Can we add the code to https://github.com/keep-network/hardhat-helpers, so we can use it in other repos?

Reply:
Opened https://github.com/keep-network/hardhat-helpers/pull/12. I'll use it here once it lands on main.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2762/commits/da8599aad54b44eeff66ae764a5376d37576f185